panel/models/tabulator.py

```diff
@@ -15,10 +15,10 @@
 from ..io.resources import bundled_files
 from ..util import classproperty
 
-JS_SRC = "https://unpkg.com/tabulator-tables@4.9.3/dist/js/tabulator.js"
-MOMENT_SRC = "https://unpkg.com/moment@2.27.0/moment.js"
+JS_SRC = "https://unpkg.com/tabulator-tables@5.0.7/dist/js/tabulator.js"
+MOMENT_SRC = "https://cdn.jsdelivr.net/npm/luxon/build/global/luxon.min.js"
```

Comment:
Shouldn't this be pinned somehow?

Reply:
Luxon is the replacement for moment.js which is THE datetime library for JS. So there will be no API changes here as that would break entire ecosystems.